hack/ci/windows.ps1

```diff
@@ -617,6 +639,15 @@ Try {
     Write-Host -ForegroundColor Green "INFO: Args: $dutArgs"
     New-Item -ItemType Directory $env:TEMP\daemon -ErrorAction SilentlyContinue  | Out-Null
 
+    # Start containerd first
+    if (-not ("$env:DOCKER_WINDOWS_CONTAINERD_RUNTIME" -eq "")) {
+        Start-Process "$env:TEMP\binary\containerd.exe" `
+                    -ArgumentList "--log-level debug" `
+                    -RedirectStandardOutput "$env:TEMP\containerd.out" `
+                    -RedirectStandardError "$env:TEMP\containerd.err"
```

Comment:
Wondering; I think we do the same for dockerd, but is it possible to combine them in a single log? (this always confused me a bit, as to why we're splitting them to separate files)

(OK to do in a follow-up in which we update both (if possible))

Reply:
Afaiu `Start-Process` does not support to redirecting those to same file.

Reply:
[Not _trivially_](https://stackoverflow.com/q/8925323/166389), but there's a few ideas in there for restructuring this to do that (mostly by not using `Start-Process`).

Reply:
> Not trivially

That's what I was afraid of.

Yes. It's definitely not urgent (and not for this PR), but I ran into the separate `.out` and `.err` files on a couple of occasions, and they're so much less useful when split.